### PR TITLE
Issue#3

### PR DESCRIPTION
**Pull request by HaiauDuong**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_
Database model for Hats [Back-End]

### CODE REVIEW

File: server/db/models/hats.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: {
+    type: Sequelize.STRING,
+    unique: true,
+    allowNull: false
+  },
+  description: {
+      type: Sequelize.TEXT,
+      allowNull: false
+  },
+  price: {
```
Comment (paolaneira) [inline server/db/models/hats.js]: Let's change this to DECIMAL or FLOAT :)

File: server/db/models/hats.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      type: Sequelize.NUMBER,
+      defaultValue: 0.00
+  },
+  quantity: {
+      type: Sequelize.NUMBER,
```
Comment (paolaneira) [inline server/db/models/hats.js]: Let's change this to INTEGER :)

File: server/db/models/hats.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  category: {
+      type: Sequelize.STRING,
+      allowNull: false
+  },
+  productImg: {
+    type: Sequelize.STRING,
+    defaultValue: "DefaultHat.jpg"
+  },
+  size: {
+      type: Sequelize.STRING,
```
Comment (paolaneira) [inline server/db/models/hats.js]: Let's use ENUM for this so that there are only specific sizes - maybe S,M,L!

**Comment by HaiauDuong** [comment]
Done editing changes

**Pull request merged**